backend/app/views/spree/admin/orders/confirm/_shipment.html.erb

```diff
@@ -44,7 +44,10 @@
       <% if order.special_instructions.present? %>
         <tr class='special_instructions'>
           <td colspan="5">
-            <strong><%= Spree.t(:special_instructions) %>:&nbsp;</strong><%= order.special_instructions %>
+            <strong>
+              <%= Spree::Order.human_attribute_name(:special_instructions) %>:&nbsp;
```

Comment:
The `&nbsp;` here previously was attempting to avoid a line break, which will no longer work now that there are additional spaces here.

I think that was unnecessary. Could we also lose the `&nbsp;` here?